src/Core/src/Platform/Android/ApplicationExtensions.cs

```diff
@@ -60,5 +62,24 @@ public static Bundle ToBundle(this IPersistedState? state)
 
 			return userInfo;
 		}
+
+		public static void UpdateNightMode(this IApplication application)
```

Comment:
Should this be called inside ThemeChanged so if the theme changes this gets updated also? Seems this way it will only run initially.

Reply:
Is invoked every time the theme changes here https://github.com/dotnet/maui/blob/17bb272bebaa227687447a44d8f54df3b0b66efc/src/Core/src/Hosting/LifecycleEvents/AppHostBuilderExtensions.Android.cs#L83

Do you mean to invoke it inside the `ThemeChanged` method? ThemeChanged fires the RequestedThemeChanged event from the .NET MAUI Application. I could move it inside but we would have to either use compiler directives or create a service or similar and use the interface from Application.